llvm/include/llvm/IR/Type.h

```diff
@@ -60,6 +60,12 @@ class Type {
     X86_FP80TyID,  ///< 80-bit floating point type (X87)
     FP128TyID,     ///< 128-bit floating point type (112-bit significand)
     PPC_FP128TyID, ///< 128-bit floating point type (two 64-bits, PowerPC)
+    DecimalFloat32TyID,
+    DecimalFloatDPD32TyID,
+    DecimalFloat64TyID,
+    DecimalFloatDPD64TyID,
+    DecimalFloat128TyID,
+    DecimalFloatDPD128TyID,
```

Comment:
I had thought we had agreed that we were just going to have d32/d64/d128 types, with the bid/dpd being datalayout-level stuff?